resources/js/v2/components/ListViewFormGroup.vue

```diff
@@ -14,8 +14,17 @@
     <div class="listview-form-group__description" v-if="$slots.description">
       <slot name="description" />
     </div>
-    <div class="listview-form-group__body">
+    <div
+      class="listview-form-group__body"
+      :class="{ 'form-append': $slots.prepend || $slots.append }"
+    >
+      <div class="form-control form-append__body" v-if="$slots.prepend">
```

Comment:
本質的に、input 前後につく要素はフォームのコントロール要素ではないので、 `form-control` にするのは無理がありそう。

今後 `form-control` のスタイルを修正した際、意図せず前後の要素のスタイルが崩れてしまうのは防ぎたいので、

```suggestion
      <div class="form-addon__body" v-if="$slots.prepend">
```

というふうにしたいです。それに合わせてCSSも修正してもらえると嬉しいです。

-----

現状でも、前後の要素に影がついてしまっており、 disabled な input 感が出てる。
（この影は無くしたい）

![image](https://user-images.githubusercontent.com/6687653/82240921-3570d880-9976-11ea-8ad5-bb61eba69a54.png)